newIDE/app/package.json

```diff
@@ -32,7 +32,7 @@
     "@blueprintjs/icons": "file:src/Utils/BlueprintJsPlaceholder",
     "@lingui/react": "git://github.com/4ian/lingui-react.git#master",
     "@material-ui/core": "4.9.10",
-    "@material-ui/icons": "4.2.1",
+    "@material-ui/icons": "4.9.1",
```

Comment:
I didn't find any issues with it, tell me if it's okay to bump up the version :)

Reply:
Yes I don't think it's too risky :)